pkg/controller/restore_controller.go

```diff
@@ -330,42 +349,62 @@ func (controller *restoreController) fetchBackup(bucket, name string) (*api.Back
 }
 
 func (controller *restoreController) runRestore(restore *api.Restore, bucket string) (warnings, errors api.RestoreResult) {
+	logContext := controller.logger.WithField("restore", kubeutil.NamespaceAndName(restore))
+
 	backup, err := controller.fetchBackup(bucket, restore.Spec.BackupName)
 	if err != nil {
-		glog.Errorf("error getting backup: %v", err)
+		logContext.WithFields(logrus.Fields{
```

Comment:
Maybe pkg/errors would help capture all the context here so it can be appended to errors.Ark rather than using the logContext as a side channel.